app/controllers/api/v3/providers_controller.rb

```diff
@@ -4,15 +4,13 @@ class ProvidersController < API::V3::ApplicationController
       before_action :build_recruitment_cycle
 
       def index
-        build_fields_for_index
-
-        @providers = if params[:search].present?
-                       return render(status: :bad_request) if params[:search].length < 2
+        if params[:search].present?
```

Comment:
I think you could just shorten this down to `if params[:search]&.length < 2` and remove the additional check on line 8.

Reply:
I thought that too but that evaluates to `if nil < 2` and errors if `params[:search]` isn't present.